.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+<!-- 👋 Hi, thanks for sending a PR to opencontributionssite! 💖.
+Please fill out all fields below and make sure each item is true and [x] checked.
+Otherwise we may not be able to review your PR. -->
+
+## PR Checklist
+
+- [ ] Addresses an existing open issue: fixes #000
+- [ ] That issue was marked as [accepting prs](https://github.com/OpenContributionsProject/opencontributionssite/issues?q=is%3Aopen+is%3Aissue+label%3A%22accepting+prs%22)
+- [ ] Steps in [CONTRIBUTING.md](https://github.com/OpenContributionsProject/opencontributionssite/blob/main/.github/CONTRIBUTING.md) were taken
+
+## Overview
+
```

Comment:
Are you open to me adding more to the pull request template kinda like  like this? -->  https://github.com/open-sauced/.github/blob/main/.github/PULL_REQUEST_TEMPLATE.md

I feel it's more beginner friendly, so we can see when people need help and stuff

Reply:
**YES** this is great! I might end up taking these changes into https://github.com/JoshuaKGoldberg/template-typescript-node-package 😄